storage.rules

```diff
@@ -0,0 +1,12 @@
+rules_version = '2';
+
+// Craft rules based on data in your Firestore database
+// allow write: if firestore.get(
+//    /databases/(default)/documents/users/$(request.auth.uid)).data.isAdmin;
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /{allPaths=**} {
+      allow read, write: if false;
+    }
+  }
+}
```

Comment:
What is this file?

Reply:
Those are the default security rules of the Firebase Storage service, handling for example our images etc. This pull request does not consider those, I only worked on the Firestore rules.